File: WebApp/Models/MI/MI_MATCLASS.cs
using System;
using System.Collections.Generic;

namespace WebApp.Models
{
    public class MI_MATCLASS : JCLib.Mvc.BaseModel
    {
        public string MAT_CLASS { get; set; }
        public string MAT_CLSNAME { get; set; }        
        public string MAT_CLSID { get; set; }
        public string CREATE_TIME { get; set; }
        public string CREATE_USER { get; set; }
        public string UPDATE_TIME { get; set; }
        public string UPDATE_USER { get; set; }
        public string UPDATE_IP { get; set; }
    }
}

File: WebApp/Scripts/app/model/MI_MATCLASS.js
Ext.define('WEBAPP.model.MI_MATCLASS', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'MAT_CLASS', type: 'string' },
        { name: 'MAT_CLSNAME', type: 'string' },
        { name: 'CREATE_TIME', type: 'string' },
        { name: 'CREATE_USER', type: 'string' },
        { name: 'UPDATE_TIME', type: 'string' },
        { name: 'UPDATE_USER', type: 'string' },
        { name: 'UPDATE_IP', type: 'string' }
    ]
});